pkg/build/controller/controller.go

```diff
@@ -102,9 +102,13 @@ func (bc *BuildController) HandleBuild(build *buildapi.Build) error {
 	}
 
 	if buildutil.IsBuildComplete(build) {
+		if _, ok := build.Annotations[buildapi.BuildCompletedAnnotation]; ok {
+			return nil
+		}
```

Comment:
:-) I couldn't think of a better name ... but basically it means that we shouldn't run the runPolicy.OnComplete command on it. Maybe `ran-policy-on-complete` ?